### PR TITLE
A11y input field 

### PR DESCRIPTION
**Pull request by svillafe**
## Why are you doing this?

In order to improve the a11y of our site, we add the behaviour of the click on the button when the user press enter or spacebar on the input field.

[**Trello Card**](https://trello.com/c/IyG06KgO/654-accessibility-issue-not-possible-to-navigate-properly-on-supporttheguardiancom-without-a-mouse)

## Screenshots
enter

### CODE REVIEW

File: assets/pages/bundles-landing/components/ContribAmounts.jsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { connect } from 'react-redux';
 
 import RadioToggle from 'components/radioToggle/radioToggle';
 import NumberInput from 'components/numberInput/numberInput';
+import { clickSubstitueKeyPressHandler } from 'helpers/utilities';
```
Comment (Ap0c) [inline assets/pages/bundles-landing/components/ContribAmounts.jsx]: `clickSubstitueKeyPressHandler` -> `clickSubstituteKeyPressHandler`?

File: assets/pages/bundles-landing/components/ContribAmounts.jsx [hunk 2]
```diff
@@ -145,6 +146,7 @@ function ContribAmounts(props: PropTypes) {
           onInput={props.userDefinedAmount}
           selected={attrs.selected}
           placeholder="Other amount (£)"
+          onKeyPress={clickSubstitueKeyPressHandler(props.onNumberInputKeyPress)}
```
Comment (Ap0c) [inline assets/pages/bundles-landing/components/ContribAmounts.jsx]: `clickSubstitueKeyPressHandler` -> `clickSubstituteKeyPressHandler`?

**Comment by svillafe** [comment]
There was an extra line when I fix a typo in 97b2ac9fdc3ba4791ac0ed53e49be4bdac3c3793. That extra line was not caught by the validator or the tests. There is a fix for this in https://github.com/guardian/support-frontend/pull/121

**Comment by svillafe** [comment]
It is working on **PROD**.

**Pull request merged**